src/main/kotlin/org/rust/cargo/toolchain/RustToolchain.kt

```diff
@@ -181,14 +158,14 @@ private fun parseRustcVersion(lines: List<String>): Version? {
 
     val major = releaseMatch.groups[1]!!.value.toInt()
```

Comment:
Yes, I also wonder if it is possible to provide a "compile time safe" accessors for regexpes. But I can't think of a language where this is possible (D perhaps?), because it would need a fair amount of compile time work.